src/lsm/manifest_log.zig

```diff
@@ -437,7 +439,6 @@ pub fn ManifestLogType(comptime Storage: type, comptime TableInfo: type) type {
                 assert(block.address > 0);
 
                 manifest_log.reading = true;
-                manifest_log.read_callback = callback;
```

Comment:
Much better—I'm surprised I did it this way, setting to `undefined` only to set the callback again in the first branch.